src/clickonce/native/projects/NetCoreCheck/EXE/NetCoreCheckExe.cpp

```diff
@@ -24,23 +28,24 @@ int __cdecl wmain(int argc, WCHAR* argv[])
     //     runtimeconfig.json file that we'll try to load.
     //     Example: NetCoreCheck.exe UseExisting Foo.runtimeconfig.json
     //  2. Otherwise we'll create a temporary runtime config file ourselves using the passed
-    //     in framework name and framework version.
-    //     Example: NetCoreCheck.exe Microsoft.WindowsDesktop.App 3.1.0
-    // In both cases the optional third parameter is the path to the log file.
+    //     in framework name and framework version, and optionally the roll forward policy.
+    //     Examples: NetCoreCheck.exe Microsoft.WindowsDesktop.App 3.1.0
+    //               NetCoreCheck.exe Microsoft.NETCore.App 3.1.0 LatestMajor
+    // In both cases the optional last parameter is the path to the log file.
```

Comment:
How do you distinguish between a roll forward policy and a log file?

Reply:
I don't, you'd have to pass a roll forward policy if you want to set a custom log file.  So you'd just pass the default.

Reply:
This might be confusing to customers. Perhaps using named arguments would be better.

Reply:
Updated, I'm now using the same parameter format as other .NET runtime utilities.